tests/db.spec.js

```diff
@@ -20,31 +21,39 @@ describe('Database', () => {
       await client.end();
     })
     describe('Users', () => {
-      let userToCreateAndUpdate, queriedUser;
-      let userCredentials = {username: 'billybob', password: 'bobbybadboy'};
+      // const userToCreateAndUpdate, queriedUser;
+      const userCredentials = {
+         firstname: "Ricky",
+          lastname: "Bobby",
+          email: "SecondIsLast@yahoo.com",
+          imgURL: "nopelol.com",
+          username: "rickybobby",
+          password: "rickybobby", 
+          isAdmin: true, 
+          address: "yee"};
       describe('createUser({ username, password })', () => {
         beforeAll(async () => {
           userToCreateAndUpdate = await createUser(userCredentials);
-          const {rows} = await client.query(`SELECT * FROM users WHERE username = $1`, [userCredentials.username]);
+          const {rows} = await client.query(`SELECT * FROM users WHERE username = $5`, [userCredentials.username]);
```

Comment:
WHERE username = $1.

The value number that is passed in is in reference to the array that follows the query.

Reply:
ok, I was worried $1 would give me .firstname not .username